### DISCUSSION TITLE
SQL Proxy cannot find certificates

### DISCUSSION THREAD

**Discussion by 3266miles** [planetscale/discussion#213] [Bug Reports] [answered]
I'm trying to use the SQL proxy as a sidecar in our Kubernetes cluster, however the both the latest tagged image, and the one I made from the v0.12.0 source (since that doesn't have a built docker image for reasons) can't seem to find a working certificate path?

![image](https://user-images.githubusercontent.com/101013130/172463671-2b6980d1-4614-4c29-a390-530d353c1da0.png)

I presume this isn't the expected behaviour of these containers?

**Comment by 3266miles** [comment, marked as answer]
Okay I heard back from support on twitter that this approach is actually deprecated and instead you should just use the `pscale` cli with the `connect` command

**Reply by clintberry** [reply]
So don't use a sidecar at all or is there a pscale cli docker image?

**Comment by 3266miles** [comment]
There is a pscale CLI image you can use yeah
